### PR TITLE
Adding laempy pull guide.

### PR DESCRIPTION
**Pull request by serundeputy**
@pirog here is how to have `laempy pull` now.

### CODE REVIEW

**Comment by pirog** [comment]
nice @serundeputy!

**Comment by serundeputy** [comment]
@pirog working out some wording in #lando channel now.

File: docs/guides/how-can-i-pull-sync-my-database-on-lamp-y-recipes.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+
+We'll need a few things to get this set up for our `laempy` apps. We need to require the `laravel/envoy` php package in our app, configure a `.env` file, configure an `Envoy.blade.php` file to define our tasks, and configure some `tooling` in our `.lando.yml` file
+. Let's jump in.
+
+## laravel/envoy
+
+To get the `laravel/envoy` package just use `composer`:
+
+```bash
+lando require laravel/envoy
```
Comment (dustinleblanc) [inline docs/guides/how-can-i-pull-sync-my-database-on-lamp-y-recipes.md]:
```suggestion
lando composer require laravel/envoy
```

**Comment by pirog** [comment]
@serundeputy merged into #2120

**Pull request closed without merging**